numgrid/becke_partitioning.cpp

```diff
@@ -0,0 +1,136 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <cmath>
+
+#include "becke_partitioning.h"
+#include "bragg.h"
+#include "error_handling.h"
+#include "parameters.h"
```

Comment:
How hard is it to modify parameters? Do I need to edit `parameters.h` and then recompile?

Reply:
I want to move some of the hardcoding to the CMake layer so that these can be set from the outside.